lib/game.rb

```diff
@@ -0,0 +1,29 @@
+require_relative './frame'
+require_relative './player'
+
+class Game
+  attr_reader :frame_number, :player_1
+
+  def initialize(player_1_name:)
+    @player_1 = Player.new(name: player_1_name)
+  end
+
+  def game_start 
+    until @player_1.frame_number == 10 do
+      p 'Put number of pins knocked down in each roll'
+      p 'if scored a strike leave roll_2 empty'
+      roll_1 = gets.chomp.to_i
```

Comment:
There is nothing stopping me entering a bad input here. I could say my first roll is 100, I would look to add a method to check if the input is valid.